content/services/development.md

```diff
@@ -75,6 +75,30 @@ summary: |
 
 {% endcall %}
 
+## Design & Strategy
+
+**We provide the full range of design services** --
+from user research & strategy
+to branding,
+from user experience & user interface design
+to integrated design systems
+and consulting around marketing & SEO.
+Deliverables include:
+logos & wordmarks,
+brand style guides,
```

Comment:
I found the repeated from/to structure hard to follow. Just an idea for cleaning that up:

```suggestion
**We provide the full range of design & branding services** --
from user research & strategy
to brand and logo design,
user experience & interface design,
integrated design systems,
and consulting around marketing & SEO.
Deliverables include:
logos & wordmarks,
brand style guides,
```